src/plugins/tile_map/public/tile_map_type.js

```diff
@@ -55,7 +55,6 @@ export function createTileMapTypeDefinition(dependencies) {
         wms: uiSettings.get('visualization:tileMap:WMSdefaults'),
       },
     },
-    requiresPartialRows: true,
```

Comment:
:information_source: This flag was never passed through the BaseVisType, so it seems this feature is no longer needed for anything and I removed the flag completely.